tests/shouldwork/Signal/RomNegative.hs

```diff
@@ -0,0 +1,33 @@
+module RomNegative where
+
+import Clash.Prelude
+import qualified Clash.Explicit.Prelude as Explicit
+import Clash.Explicit.Testbench
+
+zeroAt0
+  :: HiddenClockResetEnable dom
+  => Signal dom (Unsigned 8)
+  -> Signal dom (Unsigned 8)
+zeroAt0 a = mux en a 0
+  where
+    en = register False (pure True)
```

Comment:
We have something for this already! :-) I think this is just `ignoreFor d1` : [ignoreFor](https://hackage.haskell.org/package/clash-prelude-1.4.2/docs/Clash-Prelude-Testbench.html#v:ignoreFor). I feel inclined to make it part of `testBench` rather than `topEntity`.

Reply:
This `zeroAt0` construction is already used extensively in other tests, so it fits perfectly with its peers. Only one of the ten other instances puts it in `testBench`, despite my personal preferences ;-).